README.md

```diff
@@ -17,9 +17,9 @@ out and built.
 With Conan 2:
 
 ```
-conan create --version 1.5.8 --build=missing up-core-api/release/
-conan create --version 0.2.0 --build=missing up-cpp/release/
-conan create --version 0.2.0 --build=missing up-transport-zenoh-cpp/release/
+conan create --version 1.6.0 --build=missing up-core-api/release/
+conan create --version 1.0.0-rc0 --build=missing up-cpp/release/
+conan create --version 1.0.0-rc0 --build=missing up-transport-zenoh-cpp/release/
 conan create --version 0.1.0 --build=missing up-transport-vsomeip-cpp/release/
```

Comment:
Minor: this line could be removed since we don't currently have a release package for vsomeip